src/main/java/sirius/biz/cluster/work/DistributedTasks.java

```diff
@@ -236,11 +236,19 @@ private DistributedQueueInfo loadQueueInfo(String queueName) {
             return new DistributedQueueInfo(queueName, null, null);
         }
 
-        return new DistributedQueueInfo(queueName,
-                                        config.get("concurrencyToken").asString(),
-                                        config.get("prioritized").asBoolean() ?
-                                        Duration.ofMillis(config.getMilliseconds("penaltyTime")) :
-                                        null);
+        Duration penaltyTime = null;
+        if (config.get("prioritized").asBoolean()) {
+            long penaltyTimeMillis = config.getMilliseconds("penaltyTime");
```

Comment:
**Optional:** Constants for settings? `"prioritized"`, `"penaltyTime"` & `"concurrencyToken"`